### DISCUSSION TITLE
[List] Component supporting any external URL ?

### DISCUSSION THREAD

**Discussion by satishkumarjadhav** [adobe/aem-core-wcm-components#2150] [Q&A]
Currently the list component with Fixed List supports only internal pages with AEM, is there any plan to support adding external link ?

**Comment by tlococo** [comment]
I'm currently using the List component's Fixed List with a combination of internal pages and fully qualified URLs to external pages. The issue we have is that we can't specify a target (same tab or new tab).   Our general rule is that external links open in a new tab. 

The Text CORE component has this option.   (see attachment).   I'd like to see an enhancement to this List CORE component to accommodate this. 

![2022-04-25_12-44-25](https://user-images.githubusercontent.com/88345037/165416412-56320e98-a3a8-45c2-a790-335b3703c76d.jpg)
